adr/adr.001.folder structure/adr.001.folder structure.md

```diff
@@ -0,0 +1,260 @@
+  - `[featureName].nav.tsx`: Navigation components
+  - `[featureName].store.ts`: Data stores
+  - `[featureName].queries.ts`: API queries - using React Query
+  - `[featureName].utils.ts`: Utility functions
+  - `[featureName].data.ts`: Data models or types
+  - `[featureName].actor.ts`: State machines or actors
+  - `[featureName].client.ts`: Clients that control dependencies
+  - `[featureName].types.ts`: Types for this particular feature
+
+This naming convention offers several benefits:
+1. Easy to search: Use command+P to quickly find all 
+   relevant files for a particular domain.
+2. Clear purpose: File names immediately indicate their 
+   function within the feature.
+3. Consistent organization: Standardizes file naming 
+   across the project.
+
+*Example of the new structure for the Accounts feature:*
+
+```
+features/
+└── accounts/
+    ├── components/
+    │   ├── AccountSettingsButton.tsx
+    │   ├── ExternalWalletPicker.tsx
+    │   ├── EphemeralAccountBanner.tsx
+    │   ├── ProfileSettingsButton.tsx
+    │   └── Logout/
+    │       ├── privy.tsx
+    │       ├── privy.web.tsx
+    │       ├── wallet.tsx
+    │       └── wallet.web.tsx
+    ├── screens/
+    │   ├── Accounts.screen.tsx
+    │   ├── AccountsAndroid.screen.tsx
+    │   └── Onboarding/
+    │       ├── Onboarding.screen.tsx
+    │       ├── Onboarding.web.screen.tsx
+    │       └── components/
+    ├── navigation/
+    │   ├── Accounts.nav.tsx
+    │   ├── AccountsDrawer.nav.tsx
+    │   └── OnboardingNav.nav.tsx
+    ├── hooks/
+    │   ├── useDisconnectActionSheet.ts
+    │   └── usePhotoSelect.ts
+    ├── utils/
+    │   ├── addressBook.ts
+    │   ├── keychain/
+    │   ├── wallet.ts
+    │   └── evm/
+    ├── data/
+    │   └── store/
+    │       └── accounts.store.ts 
+    ├── accounts.utils.ts
+    ├── accounts.client.ts
+    └── accounts.queries.ts
+
+
+*Note:* Care should be taken to ensure that any platform-specific files (e.g., `.ios.tsx`, `.android.tsx`, `.web.tsx`) are correctly placed and that conditional imports are handled properly.
+
+## Migration Plan
+
+Given the size of the codebase, we will adopt a phased approach:
+
+1. **Preparation:**
+   - Communicate the plan to the development team.
+   - Freeze new feature development temporarily to reduce conflicts.
+   
+2. **Phase 1 - Critical Features:**
+   - Reorganize the most critical features first (e.g., Accounts, Conversations).
+   - Update import paths and fix any immediate issues.
+   
+3. **Phase 2 - Remaining Features:**
+   - Gradually reorganize the rest of the features.
+   - Continue updating import paths and resolving issues.
+   
+4. **Phase 3 - Shared Components and Utilities:**
+   - Move shared components and utilities into the `shared/` directory.
+   - Ensure all features are using the shared resources correctly.
+   
+5. **Testing:**
+   - Perform thorough testing after each phase.
+   - Use automated tests to catch regressions.
```

Comment:
Really don't think that thorough testing between each phase is necessary for folder changes

Reply:
I assumed these just run in our build pipelines, no?

By automated tests, I meant to include `unit`, `integration`, etc. Do we not have that run?